fix(docs/ci): update overview and qcow-download command

safl opened the pull request.

safl committed:
fix(docs/ci): update overview and qcow-download command

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

safl committed:
refactor(cijoe,gha): move 'system-imaging' into standalone config

Maintaining the repository of bootable system images, their cloud-init
source-images (when available), was scattered in the different
target-config-files. This part can benefit by reducing redudancy and
easing maintenance.

E.g. some paths did not use the "new" naming convention, thus, would not
use the right or cached image.

Additionally, then there were references to non-existing system images,
some remain but some were removed.

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

safl committed:
fix(cijoe): remove deprecated configs

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

NaddiNadja reviewed (dismissed):
very nice very clean! lgtm :)

NaddiNadja commented on docs/contributing/ci/runners/hetzner-setup/index.rst:
```diff
@@ -110,18 +116,28 @@ you must manually retrieve it from the private Hetzner Storage Box. The
+    --http-user ${HSB_USERNAME} \
+    --ask-password \
+    https://${HSB_USERNAME}.your-storagebox.de/win11.qcow2 \
+    /home/ghr/system_imaging/disk/
```
I took a look at the man for wget, and you might want to add the path as the `--directory-prefix` (or `-p`) argument instead, so:
```shell
wget \
    --http-user ${HSB_USERNAME} \
    --ask-password \
    --directory-prefix /home/ghr/system_imaging/disk/ \
    https://${HSB_USERNAME}.your-storagebox.de/win11.qcow2
```

safl replied:
nice one. thx.

safl committed:
fix(docs/ci): add command to allow vfio-pci for qemu

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

safl committed:
fix(docs/ci): add command on ghr password

Also, add ghr to password-less sudoers in the hetzner-task.

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

safl committed:
fix(docs/ci): change the flash-image

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

safl committed:
fix(docs/ci): include vnc with the qemu-build

This is very useful when troubleshooting windows guests, and guests in
general with network or config-issues.

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

safl committed:
fix(docs/ci): add virtio download as part of prep

Signed-off-by: Simon A. F. Lund <simon.lund@samsung.com>

karlowich reviewed (approved):
LGTM
